src/math/Box2.js

```diff
@@ -24,7 +33,7 @@ Object.assign( Box2.prototype, {
 
 	},
 
-	setFromPoints: function ( points ) {
+	setFromPoints( /** @type {Vector2[]} */ points ) {
```

Comment:
This is an alternative way to define the types of parameters, as alternative to `@param`.

We can choose to use one of the other, but I used both for sake of example.